app/src/forms/minesattestations/dataService.js

```diff
@@ -1,15 +1,34 @@
+const equal = require('fast-deep-equal');
 const log = require('npmlog');
 const { transaction } = require('objection');
 const { v4: uuidv4 } = require('uuid');
 
 const constants = require('./constants');
 const Models = require('./models');
 
+const copyAndRemoveStamps = (obj) => {
+  let items = obj;
+  if (!Array.isArray(items)) {
+    items = [obj];
+  }
+  return items.map(o => {
+    const x = {...o};
+    ['createdAt','createdBy','updatedAt','updatedBy'].forEach(p => delete x[p]);
+    return x;
+  });
+};
+
+const areTheseTheSame = (a, b) => {
```

Comment:
A more assertive "isIdentical" is probably better than a rhetorical question format.